src/components/Buttons.js

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import PropTypes from 'prop-types';
+
```

Comment:
- [ ] Good job making the changes @jash-debug, While you did a great job, please Consider removing disabled liters. Remember linter is here to force you to write clean code and help you identify what could possibly be improved on the current codebase.